AEPMessaging/Sources/Messaging.swift

```diff
@@ -261,6 +261,43 @@ public class Messaging: NSObject, Extension {
             return
         }
 
+        if event.isLiveActivityUpdateTokenEvent {
+            // Extract token and liveActivityID
+            guard let token = event.liveActivityUpdateToken else {
+                Log.warning(label: MessagingConstants.LOG_TAG, "Unable to process Live Activity update event (\(event.id.uuidString)) because a valid token could not be found in the event.")
+                return
+            }
+
+            guard let liveActivityID = event.liveActivityID else {
+                Log.warning(label: MessagingConstants.LOG_TAG, "Unable to process Live Activity update event (\(event.id.uuidString)) because a valid Live Activity ID could not be found in the event.")
+                return
+            }
+            sendLiveActivityUpdateToken(liveActivityID: liveActivityID, token: token, event: event)
+            return
+        }
+
+        if event.isLiveActivityStartEvent {
+            // Should not fail, since `origin` is a non-optional property on `LiveActivityData`
+            guard let origin = event.liveActivityOrigin else {
+                Log.warning(label: MessagingConstants.LOG_TAG, "Unable to process Live Activity start event (\(event.id.uuidString)) because a valid Live Activity 'origin' could not be found in the event.")
+                return
+            }
+
+            if let liveActivityID = event.liveActivityID {
+                sendLiveActivityStart(liveActivityID: liveActivityID, origin: origin, event: event)
+                return
+            }
+
+            if let channelID = event.liveActivityChannelID {
+                sendLiveActivityStart(channelID: channelID, origin: origin, event: event)
+                return
+            }
```

Comment:
do we ever have both of these?

would it be cleaner to just send both values directly from the event, and let `sendLiveActivityStart` handle situations where both ids are nil?